README.md

```diff
@@ -146,18 +171,18 @@ Description for all the available frontmatter properties
 title: Libraries 
 <!-- Small description for html metadata, if not present the first couple of paragraphs will be used instead -->
 description: Libraries used from Filecoin
-<!-- This will be used to order the navigation and any other listing of pages -->
+<!-- This will be used to order the ToC, navigation and any other listings of pages -->
 weight: 3
 <!-- This will make a page section collapse in the navigation -->
 bookCollapseSection: true
 <!-- This will hidden the page from the navigation -->
 bookhidden: true
 <!-- This is used in the dashboard to describe the importance of the page content -->
 dashboardWeight: 2
-<!-- This is used in the dashboard to describe the state of the page content options are "incorrect", "wip", "incomplete" and "stable" -->
+<!-- This is used in the dashboard to describe the state of the page content options are "missing", "incorrect", "wip", "reliable", "stable" or "n/a" -->
 dashboardState: stable
-<!-- This is used in the dashboard to describe if the theory of the page has been audited, options are 1 or 0 -->
-dashboardAudit: 1
+<!-- This is used in the dashboard to describe if the theory of the page has been audited, options are "missing", "wip", "stable" or "n/a" -->
```

Comment:
`stable` doesn't make sense here. Ideally it would be the date of the last audit. if we can't have that then `done` or `exists` would make sense.